### DISCUSSION TITLE
So excited

### DISCUSSION THREAD

**Discussion by umop3plsdn** [kannibalox/pyrosimple#17] [General]
I am soooo happy to see development on this! I realize it's a niche community but i've always enjoyed pyrocore and sad that it hadn't been updated for python3(I know HUGE undertaking). I appreciate all the work you are putting into this! Hopefully I can familiarize myself with the project to contribute in the future

**Comment by kannibalox** [comment]
Yeah, I'm glad to see interest in this has been growing. I don't have any grand designs at the moment (although an TUI in the same vein as k9s could be neat), just want to make steady progress on features and improvements, so any contributions are definitely welcome.

**Comment by MrBamse** [comment]
Yeah, this couldn't have come at a better time, you are awesome! :)
